cypress/integration/app-init.spec.js

```diff
@@ -0,0 +1,17 @@
+describe('App initialization', () => {
```

Comment:
this test fails for me when I run command `npm run test`. Please, rename this file to `app-int.e2e.js` or change your jest config